_pages/qa/gsa-links.md

```diff
@@ -9,7 +9,7 @@ title: Useful GSA links
 
 * Visit [ALOHA](http://aloha.gsa.gov/) to request and document sick or annual leave.
 
-* [CHRIS](http://www.gsa.gov/chris) provides access your personnel file online.
+* [CHRIS](http://www.gsa.gov/chris) provides access your personnel file online. You’ll get an email in week 3 or so that says that your personal record has been updated with certain actions. Take this as a cue of your account being created in the system, after which you can begin the login process.
```

Comment:
While you're in there, can you fix the typo "provides access your" -> "provides access to your"?